### ISSUE TITLE
[Item] Titanium Seal of Dalaran

### ISSUE THREAD

**Issue by Smokey7777** [Stormforge-gg/Bugtracker-Fordring#1274]
**Description**: 
https://www.wowhead.com/item=44430/titanium-seal-of-dalaran

"The on use effect of this item causes an animation of your character flipping a coin in the air, and generates two chat emotes.

Bokashi casually flips his Titanium Seal of Dalaran.

and then when you catch the coin it will randomly emote one of the following:
Bokashi finds the coin face down for tails!
or
Bokashi catches the coin heads up!

Basically it's a neater/more stylish way of doing /random 2 with a 10 second cooldown. Definitely a cool little addition to the game!"

**Current behaviour**:
You click the item and it does nothing.

**Expected behaviour**: 
It should do above.

**Revision**: 
Live, June 11th.

**Comment by Wolffenstein** [comment]
Fixed in rev 1511.
